frame/composable-traits/src/undercollateralized_loans/configs.rs

```diff
@@ -0,0 +1,299 @@
+use crate::defi::CurrencyPair;
+use frame_support::pallet_prelude::*;
+use sp_runtime::Perquintill;
+use sp_std::{
+	collections::{btree_map::BTreeMap, btree_set::BTreeSet},
+	vec::Vec,
+};
+
+#[derive(Encode, Decode, Default, TypeInfo, RuntimeDebug, Clone, Eq, PartialEq)]
+pub struct MarketConfig<AccountId, AssetId, BlockNumber, VaultId>
+where
+	AccountId: Clone + Eq + PartialEq,
+	AssetId: Clone + Eq + PartialEq,
+	BlockNumber: Clone + Eq + PartialEq,
+	VaultId: Clone + Eq + PartialEq,
+{
+	/// The account id of this market
+	account_id: AccountId,
+	/// The owner of this market.
+	manager: AccountId,
+	/// The vault containing the borrow asset.
+	borrow_asset_vault: VaultId,
+	/// The asset being used as collateral.
+	collateral_asset_id: AssetId,
+	/// The asset being used as borrow asset.
+	borrow_asset_id: AssetId,
+	/// Number of blocks until invalidate oracle's price.
+	max_price_age: BlockNumber,
+	/// Borrowers which are allowed to use the service.
+	whitelist: BTreeSet<AccountId>,
```

Comment:
This should be bounded

Reply:
@benluelo  Did not get which bound do you mean?

Reply:
The whitelist size should be bounded, see this: https://paritytech.github.io/substrate/master/frame_support/storage/bounded_btree_set/struct.BoundedBTreeSet.html

Reply:
Done bounds through inputs validation.